Jenkinsfile

```diff
@@ -7,4 +7,6 @@ dockerfile {
     upstreamProjects = 'confluentinc/rest-utils'
     nodeLabel = 'docker-oraclejdk8-compose-swarm'
     cron = ''
+    cpImages = true
+    osTypes = ['deb9', 'rhel8']
```

Comment:
are we planning to support deb9 for this change?

Reply:
@mohnishbasha This image has always used a Debian9 base image. cp-base-new has always been debian 9, which is what these images were always based on.